ext/opentelemetry-ext-opentracing-shim/CHANGELOG.md

```diff
@@ -2,7 +2,12 @@
 
 ## Unreleased
 
-- Implement extract and inject support for HTTP_HEADERS and TEXT_MAP formats.
+## 0.3a0
+
+Released 2019-12-11
+
+- Implement extract and inject support for HTTP_HEADERS and TEXT_MAP formats
+  ([#256](https://github.com/open-telemetry/opentelemetry-python/pull/290))
```

Comment:
should this reference pr#256? 290 is removing create_span.

Reply:
Yes it should! Thanks for catching that.

Reply:
Picked this change onto the release branch as well: https://github.com/open-telemetry/opentelemetry-python/commit/463dd24d8d55a3bfc1bc41b6575f10fa0f9d825a.